source/_integrations/verisure.markdown

```diff
@@ -138,3 +138,16 @@ automation:
 | disable_autolock | Disables autolock function for a specific lock. |
 | enable_autolock | Enables autolock function for a specific lock. |
 | smartcam_capture | Capture a new image from a specific smartcam. |
+
+## 2 Factor Authentication Workaround
+
+In March 2021 Verisure added new 2fa rules that aren't fully support through their third party API integration. If you have 2FA enabled (which is forced as default) You might not be able to use this integration. Here are some steps for a work around.
+
+1. Create a specific home assistant user. (not necessary but it's not recommended to turn of 2fa for your admin user).
```

Comment:
- Home Assistant is spelled wrongly.
- Write an introduction that explains the separate account (instead of a note in parenthesis)